modules/evm/src/lib.rs

```diff
@@ -876,6 +884,7 @@ impl<T: Config> Module<T> {
 	}
 
 	fn do_update_storage_usage(
```

Comment:
add `#[require_transactional]` to make sure this can only be called within transactional because it now have return error after state change.